docs/howto/deploy.md

```diff
@@ -1,4 +1,4 @@
-# Deploying the Azure Service Operator
+# Building the Azure Service Operator locally and deploying
 
 The easiest way to deploy the Azure Service Operator is to use Helm charts. Follow instructions [here](helmdeploy.md).
```

Comment:
Should probably references the prereqs.md file here somewhere? Since there are clearly some prerequisite steps (docker + go at least)

Reply:
Ah, possibly like below in the "Deploy the operator" section, put a note at the top of the "Build the operator" section:

> [!NOTE]
> This step assumes you have the prerequesites installed. See prerequisites [here](prereqs.md) for information on setup.